### PR TITLE
Issue templates for bugs and feature requests

### PR DESCRIPTION
**Pull request by bauersimon**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    attributes:
+      label: Reproducer
+      description: "How is the unexpected behavior triggered?"
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Logs
+      description: "The (reduced) output of `eval-dev-quality`, or `<result-directory>/evaluation.log`."
```
Comment (zimmski) [inline .github/ISSUE_TEMPLATE/1-bug.yaml:29]: <result-directory> -> ${result directory} please use bash-like variables consistently everywhere.

File: .github/ISSUE_TEMPLATE/3-roadmap.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Roadmap issue
+name: Roadmap Issue
```
Comment (zimmski) [inline .github/ISSUE_TEMPLATE/3-roadmap.md:2]: Please no title-case, consistently, i want to get rid of it.

File: evaluate/version.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package evaluate
 
 // Version holds the current version of the evaluation benchmark.
-var Version = "0.5.0"
+var Version = "0.6.2"
```
Comment (zimmski) [inline evaluate/version.go:4]: Please open an issue that we automate this. Release -> update, automatic.

File: .github/ISSUE_TEMPLATE/2-enhancement.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: Enhancement / Feature Request
+about: Improving upon existing behavior.
```
Comment (zimmski) [inline .github/ISSUE_TEMPLATE/2-enhancement.md:3]: "upon"? Please use simpler wording, always.

File: .github/ISSUE_TEMPLATE/2-enhancement.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+labels: enhancement
+---
+
+:exclamation: **Please help us keep the number of duplicated issues small!** Have a look at the existing [feature requests](https://github.com/symflower/eval-dev-quality/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement), our latest [roadmap](https://github.com/symflower/eval-dev-quality/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap) and [open pull requests](https://github.com/symflower/eval-dev-quality/pulls). We kindly ask you to add your suggestions/concerns there in case the feature is already being tracked.
+
+## Problem
+
+> Please describe the problem/inconvenience to be solved.
```
Comment (zimmski) [inline .github/ISSUE_TEMPLATE/2-enhancement.md:11]: Please no "problem/inconvenience " ... write a sentence this/is/not/that/readable and choices are not good. Keep it simple, simple wording.

File: evaluate/version.go (hunk 3 above)
Comment (bauersimon) [reply evaluate/version.go:4]: https://github.com/symflower/eval-dev-quality/issues/344

**Pull request merged**